DOCS/man/ao.rst

```diff
@@ -156,6 +156,9 @@ Available audio output drivers are:
         value makes the audio stream react faster, e.g. to playback speed
         changes.
 
+    ``--pipewire-remote=<remote>``
+        Specify the PipeWire remote to connect to. An empty <remote> string uses the default remote.
+
```

Comment:
I think it's worth providing more of an explanation here. A quick google search doesn't return good results for what a remote is (people think it's remote networking). so providing an example will help.

Reply:
This does indeed not support network remoting. Just the "remote" PipeWire in contrast to the one loaded into the current process.

Is the new wording better?